### ISSUE TITLE
[archetype] Child module b.tests.core of b/pom.xml does not exist

### ISSUE THREAD

**Issue by manandbytes** [caniszczyk/minerva#8]
Eclipse project generated by archetype is incorrect: build fails with

```
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project g:b:1.0-SNAPSHOT (/opt/src/b/pom.xml) has 1 error
[ERROR]     Child module /opt/src/b/b.tests.core of /opt/src/b/pom.xml does not exist
[ERROR] 
```

...but there is module _b.core.tests_.

**Comment by caniszczyk** [comment]
Thanks for the report. There's still some work that needs to be done to cleanup the archetype before it's donated to eclipse.org :)

**Comment by manandbytes** [comment]
Fixed by @613302c1c3629d84a73d
